azurerm/provider.go

```diff
@@ -280,6 +280,7 @@ func Provider() terraform.ResourceProvider {
 			"azurerm_scheduler_job_collection":                                               resourceArmSchedulerJobCollection(),
 			"azurerm_sql_database":                                                           resourceArmSqlDatabase(),
 			"azurerm_sql_elasticpool":                                                        resourceArmSqlElasticPool(),
+			"azurerm_sql2017_elasticpool":                                                    resourceArmSql2017ElasticPool(),
```

Comment:
Is this the agreed naming for this new resource ? I remember it's `azurerm_ms_sql_elasticpool`

Reply:
In a conversation with @katbyte we decided on `sql2017`